features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml

```diff
@@ -655,6 +655,14 @@
 				<AttributeID>uuid</AttributeID>
 				<Description>Claim to store the user's unique id. (UUID)</Description>
 			</Claim>
+			<Claim>
+				<ClaimURI>http://wso2.org/claims/identity/verificationPendingEmail</ClaimURI>
+				<DisplayName>Verification Pending Email</DisplayName>
+				<!-- If user store based identity store is used, a proper mapped attribute
+				in your user store must be configured for this. -->
+				<AttributeID>verificationPendingEmail</AttributeID>
+				<Description>Claim to store email address to be updated until account verification.</Description>
```

Comment:
A user's account is already verified at this stage. So here user tries to verify the channel. So I think this should be like `Claim to store newly updated email address until the new email address is verified`. WDYT?

Reply:
+1
Changed it accordingly